cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -178,8 +368,651 @@ class GameWorldService {
             });
 
         unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_class",
-            [this](const uint64_t& instance) -> uint64_t {
-                return reinterpret_cast<uint64_t>(reinterpret_cast<UObject*>(instance)->GetClass());
+            [this](const uint64_t& uobject) -> uint64_t {
+                return reinterpret_cast<uint64_t>(reinterpret_cast<UObject*>(uobject)->GetClass());
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_static_class",
+            [this](const std::string& class_name) -> uint64_t {
+                return reinterpret_cast<uint64_t>(UnrealClassRegistrar::getStaticClass(class_name));
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "spawn_actor",
+            [this](
+                const std::string& class_name,
+                const std::vector<double>& location,
+                const std::vector<double>& rotation,
+                const std::map<std::string,std::string>& unreal_obj_strings) -> uint64_t {
+
+                SP_ASSERT(location.size() == 3);
+                SP_ASSERT(rotation.size() == 3);
+
+                UnrealObj<FSpActorSpawnParameters> sp_spawn_params("SpawnParams");
```

Comment:
Let's name this parameter "SpawnParameters" because that is what it's called in the Unreal documentation.

```
UnrealObj<FSpActorSpawnParameters> sp_spawn_params("SpawnParameters");
```

https://docs.unrealengine.com/4.26/en-US/API/Runtime/Engine/Engine/UWorld/SpawnActor/4/